src/components/QRCode/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {Ref} from 'react';
+import QRCodeLibrary from 'react-native-qrcode-svg';
+import {ImageSourcePropType} from 'react-native';
+import defaultTheme from '../../styles/themes/default';
+import CONST from '../../CONST';
+
+type QRCodeProps = {
+    /** The QR code URL */
+    url: string;
+
+    /**
+     * The logo which will be displayed in the middle of the QR code.
+     * Follows ImageProps href from react-native-svg that is used by react-native-qrcode-svg.
+     */
+    logo?: ImageSourcePropType;
+
+    /** The size ratio of logo to QR code */
+    logoRatio?: typeof CONST.QR.DEFAULT_LOGO_SIZE_RATIO | typeof CONST.QR.EXPENSIFY_LOGO_SIZE_RATIO;
+
+    /** The size ratio of margin around logo to QR code */
+    logoMarginRatio?: typeof CONST.QR.DEFAULT_LOGO_MARGIN_RATIO | typeof CONST.QR.EXPENSIFY_LOGO_MARGIN_RATIO;
```

Comment:
Maybe worth to create a type represent that? `typeof CONST.QR.DEFAULT_LOGO_MARGIN_RATIO | typeof CONST.QR.EXPENSIFY_LOGO_MARGIN_RATIO`